libs/application/templates/inheritance-report/src/lib/messages.ts

```diff
@@ -737,19 +737,90 @@ export const m = defineMessages({
   },
   funeralCostTitle: {
     id: 'ir.application:funeralCostTitle',
-    defaultMessage: 'Útfarakostnaður',
+    defaultMessage: 'Útfararkostnaður',
     description: '',
   },
   funeralCostDescription: {
     id: 'ir.application:funeralCostDescription',
-    defaultMessage: 'Útfarakostnaður samkvæmt yfirlit um útfarakostnað.',
+    defaultMessage: 'Útfararkostnaður samkvæmt yfirlit um útfararkostnað.',
```

Comment:
Sounds strange. Is this supposed to be `Útfararkostnaður samkvæmt yfirliti um útfararkostnað.`?